Cloud changelog Nov 17th (#6039)
mattermost/docs: source/install/cloud-changelog.md

Latest Mattermost Cloud releases:

- [Release 2022-11-17](#release-2022-11-17)
- [Release 2022-11-10](#release-2022-11-10)
- [Release 2022-10-27](#release-2022-10-27)
- [Release 2022-10-20](#release-2022-10-20)
- [Release 2022-10-13](#release-2022-10-13)
- [Release 2022-10-06](#release-2022-10-06)

## Release 2022-11-17

### Improvements

#### User Interface (UI)

>  - The “Mark as Unread” option was added to the ``…`` menu for channels in the left-hand side sidebar. Alt+click on a channel on the left-hand side also now marks the last post in the channel as unread.

Reviewer: ```suggestion
 - The **Mark as Unread** option was added to the ``…`` menu for channels in the left-hand side sidebar. Pressing Alt while selecting a channel on the left-hand side now also marks the last post in the channel as unread.
```

 - Channel members are now able to remove themselves from a channel via the right-hand side channel members list.

>  - Updated prepackaged boards version to v7.5.0.

Reviewer: ```suggestion
 - Updated prepackaged Boards version to v7.5.0.
```

>  - Updated prepackaged Boards version to v7.5.0.

Reviewer 2: @amyblais Did we not want to include the list of Board improvements as well?

Author: Yes, indeed, added now.

>  - Updated prepackaged Calls version v0.10.0.

Reviewer: ```suggestion
 - Updated prepackaged Calls version to v0.10.0.
```

 - Downgraded Bulgarian, Persian, and Simplified Chinese language support to Alpha.

#### Administration
 - The **System Console > File Sharing and Download** setting is now unhidden and honored in Cloud.
 - **Total Active Users** was renamed to **Total Activated Users** in **system console > Site Statistics**.
 - Optimized ``ThreadStore.MarkAllAsUnreadByTeam``.

### API Changes

>  - A new API method ``RegisterCollectionAndTopic(collectionType, topicType string) (error)`` was added  to the Plugin API and the following hooks:

Reviewer: ```suggestion
 - A new API method ``RegisterCollectionAndTopic(collectionType, topicType string) (error)`` was added to the Plugin API and the following hooks:
```

    - ``UserHasPermissionToCollection(c *Context, userID, collectionType, collectionId string, permission *model.Permission) (bool, error)``
    - ``GetAllCollectionIDsForUser(c *Context, userID, collectionType string) ([]string, error)``
    - ``GetAllUserIdsForCollection(c *Context, collectionType, collectionID string) ([]string, error)``
    - ``GetTopicRedirect(c *Context, topicType, topicID string) (string, error)``
    - ``GetCollectionMetadataByIds(c *Context, collectionType string, collectionIds []string) (map[string]model.CollectionMetadata, error)``
    - ``GetTopicMetadataByIds(c *Context, topicType string, topicIds []string) (map[string]*model.TopicMetadata, error)``

### Bug Fixes
 - Fixed an issue where exports did not contain favorited Direct Message channels.
 - Fixed an issue where screen readers did not announce search results on "Invite members to channel" modal.
 - Fixed an issue where screen readers did not announce the status of the user when hovering over the user status icon.

>  - Fixed an issue where users with narrow screens could not see the Profile Setting section.

Reviewer: ```suggestion
 - Fixed an issue where users with narrow screens could not see the **Profile Settings** section within the **Settings** modal.
```

>  - Fixed an issue where users were unable to access the create account button on narrow screens.

Reviewer: ```suggestion
 - Fixed an issue where users were unable to access the **Create an account** option on narrow screens.
```

 - Downgraded Bulgarian, Persian, and Simplified Chinese language support to Alpha.

#### Administration
 - The **System Console > File Sharing and Download** setting is now unhidden and honored in Cloud.
 - **Total Active Users** was renamed to **Total Activated Users** in **system console > Site Statistics**.
 - Optimized ``ThreadStore.MarkAllAsUnreadByTeam``.

### API Changes
 - A new API method ``RegisterCollectionAndTopic(collectionType, topicType string) (error)`` was added to the Plugin API and the following hooks:
    - ``UserHasPermissionToCollection(c *Context, userID, collectionType, collectionId string, permission *model.Permission) (bool, error)``
    - ``GetAllCollectionIDsForUser(c *Context, userID, collectionType string) ([]string, error)``
    - ``GetAllUserIdsForCollection(c *Context, collectionType, collectionID string) ([]string, error)``
    - ``GetTopicRedirect(c *Context, topicType, topicID string) (string, error)``
    - ``GetCollectionMetadataByIds(c *Context, collectionType string, collectionIds []string) (map[string]model.CollectionMetadata, error)``

>     - ``GetTopicMetadataByIds(c *Context, topicType string, topicIds []string) (map[string]*model.TopicMetadata, error)``

Author: @lieut-data Are they behind a feature flag?

>     - ``GetTopicMetadataByIds(c *Context, topicType string, topicIds []string) (map[string]*model.TopicMetadata, error)``

Reviewer 3: @amyblais, it's not possible to put the API endpoints themselves behind a feature flag, but they are indeed non-functional without that feature flag.
